### PR TITLE
uses native errors instead of github.com/pkg/errors

### PR DESCRIPTION
**Pull request by muma378**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

This PR removes "github.com/pkg/errors" and uses the go native "errors" pkg.

#### Which issue(s) this PR fixes:

Fixes a part of #103043

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test
/lgtm
/assign @dims 
Thanks

File: test/utils/pki_helpers.go [hunk 1]
```diff
@@ -23,12 +23,11 @@ import (
 	"crypto/x509"
 	"crypto/x509/pkix"
 	"encoding/pem"
+	"errors"
 	"math"
 	"math/big"
 	"time"
 
-	"github.com/pkg/errors"
```
Comment (aojea) [inline test/utils/pki_helpers.go:30]: you have to replace Lines 61 and 64 before removing the imported library

**Comment by aojea** [comment]
/lgtm cancel
current PR it is wrong

File: test/utils/pki_helpers.go (hunk 1 above)
Comment (muma378) [reply test/utils/pki_helpers.go:30]: thx! replaced now, feel sorry about the mistake 😅

**Comment by wzshiming** [comment]
Please edit `release-note`
> \```release-note
> NONE
> \```

File: test/utils/pki_helpers.go (hunk 1 above)
Comment (aojea) [reply test/utils/pki_helpers.go:30]: no worries, that happens to all of us

**Pull request merged**